spec/fixtures/vcr_cassettes/viewing_a_record/when_the_record_has_an_IIIF_manifest_URL/renders_the_Mirador_viewer_with_no_workspace_type.yml

```diff
@@ -94,4 +94,106 @@ http_interactions:
       encoding: ASCII-8BIT
       string: var _GBSBookInfo = {};
   recorded_at: Thu, 25 May 2023 18:31:49 GMT
+- request:
+    method: get
```

Comment:
I wonder if we should do what we do In scholarsphere for this? 

https://github.com/psu-libraries/scholarsphere/blob/main/bin/ci-rspec#L54

I'm not sure we should be recording calls to get chromedriver, we also may want to allow chromedriver.storage.googleapis.com in our allowed hosts configuration?

Reply:
Oh i see you did revert this change.